packages/edit-site/src/components/global-styles/screen-block.js

```diff
@@ -232,6 +241,19 @@ function ScreenBlock( { name, variation } ) {
 		setStyle( { ...newStyle, border: { ...updatedBorder, radius } } );
 	};
 
+	const onChangeBackground = ( newStyle ) => {
```

Comment:
TODO: Also need to set the default-background size/position in the incoming style object if it's not already

Reply:
Maybe in packages/block-editor/src/components/global-styles/use-global-styles-output.js

Reply:
See https://github.com/WordPress/gutenberg/pull/60405/files#diff-52acb55f718dce035b748387f60ef068cd3769674b553584f7df6f790e9fa84eR397